1. Basic Web Development/script.js

```diff
@@ -1,90 +1,70 @@
-const API_URL = "http://localhost:3000";
-const HEADERS = {
-    "Content-Type": "application/json",
+var API_URL = "http://localhost:3000";
```

Comment:
why var?

Reply:
This is how the compiler translated it from TS.